tmt/log.py

```diff
@@ -331,7 +333,7 @@ def filter(self, record: logging.LogRecord) -> bool:
         if message_verbosity_level is None:
             return True
 
-        return True if details['logger_verbosity_level'] >= message_verbosity_level else False
+        return details['logger_verbosity_level'] >= message_verbosity_level
```

Comment:
Now this is weird, there's a change down below:

```diff
-            return True if state == 'running' else False
+            return bool(state == "running")
```

Any idea why `bool()` is not applied here as well?

Reply:
The `bool()` is there because state is `any`. There is a comment above it.